docs/deployment/cli.md

```diff
@@ -23,664 +23,303 @@ specific language governing permissions and limitations
+Options for both the above setups:
+        [--backend <file|rocks>]
+        [--checkpoint-dir <filepath>]
+        [--async-checkpoints <true|false>]
+        [--incremental-checkpoints <true|false>]
+        [--output <filepath> OR null for stdout]
+
+Using standalone source with error rate 0.000000 and sleep delay 1 millis
+
+Job has been submitted with JobID 97b20a0a8ffd5c1d656328b0cd6436a6
+```
 
-Disposes the savepoint at the given path. The savepoint path is returned by the savepoint trigger command.
+See how the command is equal to the [initial run command](#submitting-a-job) except for the 
+`--fromSavepoint` parameter which is used to refer to the state of the 
+[previously cancelled job](#stopping-a-job-gracefully-creating-a-final-savepoint). A new JobID is 
+generated that can be used to maintain the job.
 
-If you use custom state instances (for example custom reducing state or RocksDB state), you have to specify the path to the program JAR with which the savepoint was triggered in order to dispose the savepoint with the user code class loader:
+By default, we try to match the whole savepoint state to the job being submitted. If you want to 
+allow to skip savepoint state that cannot be restored with the new job you can set the 
+`--allowNonRestoredState` flag. You need to allow this if you removed an operator from your program 
+that was part of the program when the savepoint was triggered and you still want to use the savepoint.
 
 {% highlight bash %}
-./bin/flink savepoint -d <savepointPath> -j <jarFile>
+$ ./bin/flink run \
+      --fromSavepoint <savepointPath> \
+      --allowNonRestoredState ...
 {% endhighlight %}
+This is useful if your program dropped an operator that was part of the savepoint.
 
-Otherwise, you will run into a `ClassNotFoundException`.
+{% top %}
 
-## Usage
+## CLI Actions
+
+Here's an overview of actions supported by Flink's CLI tool:
+<table class="table table-bordered">
+    <thead>
+        <tr>
+          <th class="text-left" style="width: 25%">Action</th>
+          <th class="text-left" style="width: 50%">Purpose</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code class="highlighter-rouge">run</code></td>
+            <td>
+                This action executes jobs. It requires at least the jar containing the job. Flink-
+                or job-related arguments can be passed if necessary.
+            </td>
+        </tr>
+        <tr>
+            <td><code class="highlighter-rouge">run-application</code></td>
+            <td>
+                This action executes jobs in <a href="{% link deployment/index.md %}#deployment-modes">
+                Application Mode</a>. Other than that, it requires the same parameters as the 
+                <code class="highlighter-rouge">run</code> action.
+            </td>
+        </tr>
+        <tr>
+            <td><code class="highlighter-rouge">info</code></td>
```

Comment:
This command hasn't been mentioned explicitly in the sections above. Was this intentional?

Reply:
I considered the `info` action being for more advanced users. The CLI job lifecycle management addresses beginners instead. That was the reason I didn't add it to the `Job Lifecycle Management` section.

Reply:
Ok, sounds good to me. I think that this command is also not used very often.